syntax = "proto3";

import "google/protobuf/any.proto";
import "common/constant/ResponseMsgType.proto";
import "common/constant/RequestMsgType.proto";
option java_package = "com.huasheng.sa.quant.open.sdk.protobuf.common.msg";
option java_outer_classname = "ResponseProto";
message PBResponse {
	ResponseMsgType responseMsgType = 1;
	string requestId = 2;
	int32 responseCode = 3;
	string responseMsg = 4;
	google.protobuf.Any payload = 5; // Any对象
	RequestMsgType requestMsgType = 6;
	string sign = 7;
	int64  responseTime = 8;
}